AppKit/CPImage.j

```diff
@@ -161,7 +164,7 @@ function CPAppKitImage(aFilename, aSize)
 - (id)initByReferencingFile:(CPString)aFilename size:(CGSize)aSize
 {
     // Quietly return nil like in Cocoa, rather than crashing later.
-    if (aFilename == nil)
+    if (aFilename === undefined || aFilename === nil)
```

Comment:
Please use `aFilename == nil` instead of `aFilename === undefined || aFilename === nil` as it is equivalent and much more clean looking